releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -312,6 +312,7 @@ private static ReleaseNotesMessage createReleaseNotesMessage(int number, String
         final Field titleField = issue.getClass().getDeclaredField("title");
         titleField.setAccessible(true);
         titleField.set(issue, title);
+        escape(issue, titleField);
```

Comment:
you should not escape all other formats, it is problem only for Github release page.

Reply:
Changes Done

Reply:
@romani ping